### PR TITLE
[FLINK-14115][docs-zh] Translate DataStream Code Walkthrough to Chinese

### PR DESCRIPTION
**Pull request by gaofeilong198810**
## What is the purpose of the change

Translate DataStream Code Walkthrough to Chinese

## Brief change log

Translate DataStream Code Walkthrough to Chinese

## Verifying this change

I verified this change by running cmd 'build_docs.sh -i' and it's worked

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

**Comment by gaofeilong198810** [comment]
I found that flinkbot marked my PR as 'FAILURE Build', is there anything incorrect in my PR?

File: docs/getting-started/walkthroughs/datastream_api.zh.md [hunk 1]
```diff
@@ -490,13 +491,13 @@ class FraudDetector extends KeyedProcessFunction[Long, Transaction, Alert] {
...
+它提供了三个用于交互的方法。`update` 用于更新状态，`value` 用于获取状态值，还有 `clear` 用于清空状态。
+如果一个 key 还没有状态，例如当程序刚启动或者调用过 `ValueState#clear` 方法时，`ValueState#value` 将会返回 `null`。
+如果需要更新状态，需要调用 `ValueState#update` 方法，直接更改 `ValueState#value` 的返回值可能不会被系统识别。
+容错处理将在 Flink 后台自动管理，你可以像与常规变量那样与状态变量进行交互。
 
-Below, you can see an example of how you can use a flag state to track potential fraudulent transactions.
+下边的示例，说明了如何使用标记状态来追踪可能的欺诈交易行为。
 
 <div class="codetabs" markdown="1">
 <div data-lang="java" markdown="1">
```
Comment (KarmaGYZ) [inline docs/getting-started/walkthroughs/datastream_api.zh.md]: We may remove the useless indentation in zh version.

File: docs/getting-started/walkthroughs/datastream_api.zh.md [hunk 2]
```diff
@@ -422,38 +422,39 @@ class FraudDetector extends KeyedProcessFunction[Long, Transaction, Alert] {
...
 
-For the first version, the fraud detector should output an alert for any account that makes a small transaction immediately followed by a large one. Where small is anything less than $1.00 and large is more than $500.
-Imagine your fraud detector processes the following stream of transactions for a particular account.
+我们先实现第一版报警程序，对于一个账户，如果出现小于 $1 美元的交易后紧跟着一个大于 $500 的交易，就输出一个报警信息。
+
+假设你的欺诈检测器所处理的交易数据如下：
 
 <p class="text-center">
-    <img alt="Transactions" width="80%" src="{{ site.baseurl }}/fig/fraud-transactions.svg"/>
+    <img alt="Transactions" width="80%" src="{{ site.baseurl }}/zh/fig/fraud-transactions.svg"/>
```
Comment (KarmaGYZ) [inline docs/getting-started/walkthroughs/datastream_api.zh.md]: I couldn't find it in code path.

File: docs/getting-started/walkthroughs/datastream_api.zh.md [hunk 3]
```diff
@@ -564,31 +565,31 @@ Below, you can see an example of how you can use a flag state to track potential
...
-* If the flag is ever cleared the timer should be canceled.
+* 当标记状态被设置为 `true` 时，设置一个在当前时间一分钟后触发的定时器。
+* 当定时器被触发时，重置标记状态。
+* 当标记状态被重置时，删除定时器。
 
-To cancel a timer, you have to remember what time it is set for, and remembering implies state, so you will begin by creating a timer state along with your flag state.
+要删除一个定时器，你需要记录这个定时器的触发时间，这同样需要状态来实现，所以你需要在标记状态后也创建一个记录定时器时间的状态。
 
 <div class="codetabs" markdown="1">
 <div data-lang="java" markdown="1">
```
Comment (KarmaGYZ) [inline docs/getting-started/walkthroughs/datastream_api.zh.md]: We may remove the useless indentation in zh version.

**Comment by gaofeilong198810** [comment]
@KarmaGYZ thanks very much for you comment, but I'm still don't understanding where the 'useless indentation' is. The code in ZH version and EN version are looking same to me. Can you help me to pointing out which exactly indentation should be removed? thx a lot.

**Comment by KarmaGYZ** [comment]
@gaofeilong198810 Yes, but the indentation in EN version seems useless. I don't think we need to sync with EN version in this point.

**Comment by gaofeilong198810** [comment]
@flinkbot run travis

**Comment by gaofeilong198810** [comment]
I have made some currection of my implement.

**Comment by gaofeilong198810** [comment]
@flinkbot run travis

**Comment by gaofeilong198810** [comment]
> @gaofeilong198810 Yes, but the indentation in EN version seems useless. I don't think we need to sync with EN version in this point.

I still don't get where the useless indentation is.

**Comment by gaofeilong198810** [comment]
@flinkbot run travis

File: docs/getting-started/walkthroughs/datastream_api.zh.md [hunk 4]
```diff
@@ -911,10 +912,10 @@ class FraudDetector extends KeyedProcessFunction[Long, Transaction, Alert] {
 </div>
 </div>
 
-### Expected Output
+### 输出结果
```
Comment (wuchong) [inline docs/getting-started/walkthroughs/datastream_api.zh.md]:
```suggestion
### 期望的结果
```

**Comment by gaofeilong198810** [comment]
@wuchong thanks for your instructions, I have finish the suggestion you provide. please take a look.

**Pull request merged**